ansible/inventory/testing/group_vars/discourse/vars.yml

```diff
@@ -0,0 +1,17 @@
+dis_hostname: testing.privacylx.org
+dis_smtp_host: mail.privacylx.org
+dis_smtp_port: 587
+dis_smtp_user: cafe-testing@privacylx.org
+dis_smtp_pass: !vault |
+      $ANSIBLE_VAULT;1.1;AES256
+      33653237373332353963346638396535343361343763633361353732393061613535373930633835
+      3339343737643734636534343934653364333533653436360a303139323561643765646262623935
+      34363730626262653931613730343831393665366134396462306539303430346362383336303836
+      6532363130323930640a336133323963646235343338303237613134343435623131313362356662
+      31393765626432636639633432316661333533343061333735303839613337396337323666623862
+      32386630333839393864613166616537343437376332656262306161636230316262393162323762
+      34636363373930373038383930376165633765353432636264663630306265366535633632333037
+      35356366313237356539
+dis_le_email: admin@privacylx.org
+dis_developer_emails:
+  - admin@privacylx.org
```

Comment:
This will be the admin user of Discourse take care not to mix things and confuse the testing with production environment in your tests.

Reply:
Spot on. I've replaced it now.